src/projective.jl

```diff
@@ -339,8 +339,26 @@ function imagept2plane(C::Camera, xy::Array, planeP::Vector, planeN::Vector)
     # Subtract the tangential distortion components and divide by the radial
     # distortion factor to get an approximation of the undistorted normalised
     # image coordinates (with no skew)
-    x_n = (x_d .- dtx)./r_d
-    y_n = (y_d .- dty)./r_d
+
+    # inverse of the radial distortion is calculated by iteration
```

Comment:
Perhaps add a reference to the paper in the comment here so that when someone else comes along in the future they know the origin of the approach taken